Dockerfile

```diff
@@ -23,10 +29,16 @@ RUN git checkout ${GENIE_VERSION}
 RUN yarn link thingtalk
 RUN yarn install
 
-COPY lib.sh generate-dataset-job.sh train-job.sh evaluate-job.sh .
+COPY lib.sh generate-dataset-job.sh train-job.sh evaluate-job.sh ./
 
 # add user genie-toolkit
 RUN useradd -ms /bin/bash -r genie-toolkit
 USER genie-toolkit
 WORKDIR /home/genie-toolkit
 
+USER root
+RUN chmod -R 755 /usr/local/share/genienlp/embeddings
```

Comment:
Same as this one really

Reply:
yeah. without this line user "genie-toolkit" could not read from that path...

Reply:
Yeah it just needs to be in the common image instead